lib/hocon/impl/abstract_config_value.rb

```diff
@@ -60,6 +157,78 @@ def require_not_ignoring_fallbacks
     end
   end
 
+  def construct_delayed_merge(origin, stack)
+    # TODO: this might not work because ConfigDelayedMerge inherits
```

Comment:
oh boy!

how do you expect this to fail, if it does?  i guess what I'm really asking is, would you expect the error to be fairly obvious or not?